### PR TITLE
Updates to Brexit actions

### PR DESCRIPTION
**Pull request by sihugh**
Imported by running:
`rake brexit_checker:convert_csv_to_yaml:actions_from_google_drive`

https://trello.com/c/JrxeXfA6/549-action-batch-update-october-x

:warning: This application is Continuously Deployed: :warning:

- Merged changes are automatically deployed to staging and production.

- Make sure you follow [the guidance for deployments](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) **before** you merge.

- Check your branch is being deployed in the [Release app](https://release.publishing.service.gov.uk/applications/finder-frontend), after merging.

### CODE REVIEW

File: app/lib/brexit_checker/actions.yaml [hunk 1]
```diff
@@ -1479,25 +1485,27 @@ actions:
...
+  title: Get approval and product authorisations for new biocidal active substances
+    for the GB market from 1 January 2021
+  consequence: You will not be able to place new biocidal active substances on the
+    GB market without approval.
   guidance_prompt: More information
-  guidance_link_text: 'Brexit: Chemicals industry guidance'
-  guidance_url: https://www.hse.gov.uk/brexit/chemicals-brexit-guidance.htm?utm_source=gov.uk&utm_medium=referral&utm_campaign=eu-transition&utm_content=checker-tool
+  guidance_link_text: 'Biocides: What you''ll need to do after the transition period
+    ends'
+  guidance_url: https://www.hse.gov.uk/brexit/biocides.htm?utm_source=gov.uk&utm_medium=referral&utm_campaign=eu-transition&utm_term=biocides&utm_content=checker-tool'
```
Comment (brucebolt) [inline app/lib/brexit_checker/actions.yaml:1508]: There's a single quote mark at the end of this line.  Not sure if it should be there?

Comment (brucebolt) [reply app/lib/brexit_checker/actions.yaml:1508]: I don't know if it's related to the `you''ll` on line 1506?

Comment (sihugh) [reply app/lib/brexit_checker/actions.yaml:1508]: Good spot, ta!

Comment (sihugh) [reply app/lib/brexit_checker/actions.yaml]: I think the one on the line above is yaml escaping. It looks OK in the [review app](https://finder-front-updates-la-d4n0cv.herokuapp.com/transition-check/results?c%5B%5D=chemical&c%5B%5D=import-from-eu&c%5B%5D=export-to-eu&c%5B%5D=owns-operates-business-organisation-uk&c%5B%5D=owns-operates-business-organisation&c%5B%5D=travel-eu-business-no&c%5B%5D=working-uk&c%5B%5D=living-uk&c%5B%5D=nationality-uk). 

The link one looks like a copy/paste issue and has been fixed in the source spreadsheet now

Comment (sihugh) [reply app/lib/brexit_checker/actions.yaml]: I've pushed the fix

**Review by brucebolt — APPROVED** [review]

**Comment by sihugh** [comment]
Thanks @brucebolt, much appreciated 👍

**Pull request merged**